flight/mcu/lib/teensyShared/rocketFSM.h

```diff
@@ -12,6 +12,14 @@ class rocketFSM {
 
    private:
     pointers *pointer_struct;
+    float linear_acceleration;
+    FSM_State rocket_state;
+    uint32_t launch_time;
```

Comment:
**To summarize the changes that I think are required:**
- Change the member variables to pointers 
- Set the member variables in the constructor, not local variables
- Rename member variables to follow convention

Reply:
Got it. Thanks for the feedback!